### PR TITLE
Quiz functionality

### PR DESCRIPTION
**Pull request by deekshithmd**
Added functionality to category page, quiz page and result page. Please review it and provide your comments

### CODE REVIEW

File: src/components/Result/Result.jsx [hunk 1]
```diff
@@ -1,17 +1,40 @@
...
 import { useNavigate } from "react-router-dom";
+import { useData } from "../../contexts";
+import { useState, useEffect } from "react";
 export const Result = () => {
+  const { data } = useData();
   const navigate = useNavigate();
+  const [score, setScore] = useState(0);
+  //const [p,setP]=useState(0)
+  // console.log("marked", data.markedAnswers);
+  // console.log("correct", data.correctAnswers);
```
Comment (sanjitsarkar) [inline src/components/Result/Result.jsx]: Remove commented code and console logging

**Review by sanjitsarkar — APPROVED** [review]

File: src/components/Result/Result.jsx (hunk 1 above)
Comment (deekshithmd) [reply src/components/Result/Result.jsx]: Ok, Thank you for pointing it out...

File: src/index.css [hunk 2]
```diff
@@ -1,5 +1,6 @@
 @import url('https://fonts.googleapis.com/css2?family=Oleo+Script:wght@700&display=swap');
 @import url("../src/components/Home/home.css");
+@import url("../src//components//Quiz/quiz.css");
```
Comment (mayuresh-tech) [inline src/index.css]: Maybe // can be replaced with /

**Review by mayuresh-tech — APPROVED** [review]
Good to merge

**Review by NamrataRaikwar2002 — APPROVED** [review]
- UI looks nice 
- functionality is working fine
Great work
one suggestion is, in quiz radio options, you can give styling to label tag to get the same style as div above label, so that you can onchange radio input by clicking on the the label as a div instead of text itself, for better UX.

Screenshot from 2022-05-04 23-08-55

File: src/index.css (hunk 2 above)
Comment (deekshithmd) [reply src/index.css]: Ok

**Comment by deekshithmd** [comment]
> * UI looks nice
> * functionality is working fine
>   Great work
>   one suggestion is, in quiz radio options, you can give styling to label tag to get the same style as div above label, so that you can onchange radio input by clicking on the the label as a div instead of text itself, for better UX.
> 
> Screenshot from 2022-05-04 23-08-55

Ok, I will check that. Thank you for suggesting

**Pull request merged**